OTAnalytics/plugin_ui/customtkinter_gui/frame_filter.py

```diff
@@ -461,18 +461,29 @@ def _get_widgets(self) -> None:
         )
 
     def _place_widgets(self) -> None:
-        self.name_label.grid(row=0, column=0, padx=PADX, pady=PADY, sticky=STICKY_WEST)
-        self.date_entry.grid(row=0, column=1, padx=PADX, pady=PADY, stick=STICKY)
-        self.hour_entry.grid(row=0, column=2, padx=PADX, pady=PADY, stick=STICKY)
-        ColonLabel(master=self).grid(row=0, column=3)
-        self.minute_entry.grid(row=0, column=4, padx=PADX, pady=PADY, stick=STICKY)
-        ColonLabel(master=self).grid(row=0, column=5)
-        self.second_entry.grid(row=0, column=6, padx=PADX, pady=PADY, stick=STICKY)
         if self._place_validation_below:
```

Comment:
Why two versions?

Reply:
In OTAnalytics, we need all ui elements in a row, because we show two times the same widget (e.g. in filter as from and to dates). In OTConfig, we want to have the widgets aligned with lower horizontal space requirements. Therefore, the columns and rows of the widgets differs.